dask_cloudprovider/gcp/instances.py

```diff
@@ -146,9 +148,9 @@ def create_gcp_config(self):
             },
             "labels": {"container-vm": "dask-cloudprovider"},
             "scheduling": {
-                "preemptible": "false",
+                "preemptible": ("true" if self.preemptible else "false"),
```

Comment:
What do you think about making the default  `preemptible=False` rather than None ?  I think keeping the massive dictionary cleaner is a good goal

Reply:
Fixed